osu.Server.Queues.ScoreStatisticsProcessor.Tests/PerformanceProcessorTests.cs

```diff
@@ -287,6 +287,19 @@ public void LegacyScoreDoesProcess()
             });
         }
 
+        [Fact]
+        public void ModsWithSettingsAreDisallowed()
+        {
+            var mods = new Mod[]
+            {
+                new OsuModDoubleTime { SpeedChange = { Value = 1.1 } },
+                new OsuModClassic { NoSliderHeadAccuracy = { Value = false } }
```

Comment:
Generally this particular assert isn't doing much in the context of settings being default or not; `OsuModClassic` is failing the `AllModsValidForPerformance()` check here because the `SoloScoreInfo` is not a legacy score.

I'd probably add another case here.

Reply:
Added a case for flashlight. Kept the classic mod case because it can't hurt.